src/server/misc/cmds/cmds.go

```diff
@@ -0,0 +1,128 @@
+package cmds
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/cmdutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/promutil"
+	"github.com/spf13/cobra"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+func Cmds() []*cobra.Command {
+	var commands []*cobra.Command
+
+	var d net.Dialer
+	setupControl(&d)
+
+	r := new(net.Resolver)
+	d.Resolver = r
+	r.Dial = func(ctx context.Context, network, address string) (_ net.Conn, retErr error) {
+		defer log.Span(ctx, "DialDNS", zap.String("network", network), zap.String("address", address))(log.Errorp(&retErr))
+		return d.DialContext(ctx, network, address) //nolint:wrapcheck
+	}
+
+	t := http.DefaultTransport.(*http.Transport).Clone()
+	t.DialContext = d.DialContext
+	t.TLSClientConfig.VerifyConnection = func(cs tls.ConnectionState) error {
```

Comment:
Just confirmed that this is fired regardless of `InsecureSkipVerify` ✅

Reply:
Yup, and not fired on plaintext HTTP connections.